### PR TITLE
SPRING-INX IITM RECIPE

### PR DESCRIPTION
**Pull request by arjun-gangwar**
## What?

Added SPRING-INX Recipe with specified guidelines mentioned in CONTRIBUTING.md

### CODE REVIEW

File: egs2/TEMPLATE/asr1/utils/utils [hunk 1]
```diff
@@ -0,0 +1 @@
+utils/
```
Comment (ftshijt) [inline egs2/TEMPLATE/asr1/utils/utils:1]: Is this used somewhere?

**Comment by sw005320** [comment]
@arjun-gangwar, thanks a lot!
Can you follow the other [README.md](https://github.com/espnet/espnet/blob/master/egs2/librispeech_100/asr1/README.md) styles and add the required information generated by `README.md` in your experimental directory?
Can you also upload a model to the HF hub and put the model link?

See https://github.com/espnet/espnet/blob/master/CONTRIBUTING.md#132-espnet2-recipes

File: egs2/spring_speech/db.sh [hunk 2]
```diff
@@ -0,0 +1,342 @@
+# Set the path of your corpus
```
[the remark this answers is not in the document]
Comment (arjun-gangwar) [reply egs2/spring_speech/db.sh]: I have removed db.sh from our recipe and added a symlink.

File: egs2/TEMPLATE/asr1/utils/utils (hunk 1 above)
Comment (arjun-gangwar) [reply egs2/TEMPLATE/asr1/utils/utils]: No, this was a mistake. I have fixed it.

**Comment by arjun-gangwar** [comment]
> @arjun-gangwar, thanks a lot! Can you follow the other [README.md](https://github.com/espnet/espnet/blob/master/egs2/librispeech_100/asr1/README.md) styles and add the required information generated by `README.md` in your experimental directory? Can you also upload a model to the HF hub and put the model link?
> 
> See https://github.com/espnet/espnet/blob/master/CONTRIBUTING.md#132-espnet2-recipes

Hi @sw005320,
Thank You for your help. At the moment, we are facing pressure due to some deadlines. We will update the recipe README.md and upload the models on Hugging Face in about a week's time.

**Comment by sw005320** [comment]
> > @arjun-gangwar, thanks a lot! Can you follow the other [README.md](https://github.com/espnet/espnet/blob/master/egs2/librispeech_100/asr1/README.md) styles and add the required information generated by `README.md` in your experimental directory? Can you also upload a model to the HF hub and put the model link?
> > See https://github.com/espnet/espnet/blob/master/CONTRIBUTING.md#132-espnet2-recipes
> 
> Hi @sw005320, Thank You for your help. At the moment, we are facing pressure due to some deadlines. We will update the recipe README.md and upload the models on Hugging Face in about a week's time.

OK.
I just merged the PR, but it would be great if you could make a follow-up PR to include them.

**Pull request merged**